packages/gnucobol/tmploc.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/libcob/common.c b/libcob/common.c
+--- a/libcob/common.c
++++ b/libcob/common.c
+@@ -5973,9 +5974,9 @@ cob_gettmpdir (void)
+ #else
+ 	 && (tmpdir = check_valid_env_tmpdir ("TMP")) == NULL
+ 	 && (tmpdir = check_valid_env_tmpdir ("TEMP")) == NULL) {
+-		if (!check_valid_dir ("/tmp")) {
+-			tmp = cob_fast_malloc (5U);
+-			strcpy (tmp, "/tmp");
++		if (!check_valid_dir ("@TERMUX_PREFIX@/tmp")) {
++			tmp = cob_fast_malloc (strlen("@TERMUX_PREFIX@") + 5U);
++			strcpy (tmp, "@TERMUX_PREFIX@/tmp");
```

Comment:
> I think it's a pitty.
> if upstream gives this fallback option, why not on termux too?
> it's definitely and legitimately termux related.

Let's move the discussion of the `/tmp` patch to the patch in question.
I think this is good, and should be kept.
This is straight forward case of fixing a hardcoded path, it just so happens that `$TMPDIR` already does this job.